pkg/audit/manager.go

```diff
@@ -425,6 +432,19 @@ func (am *Manager) writeAuditResults(ctx context.Context, resourceList []schema.
 	return nil
 }
 
+func (am *Manager) skipExcludedNamespace(namespace string) bool {
+	am.operationExcluder.Mux.Lock()
```

Comment:
Locking/unlocking should happen inside of methods of operationExcluder. This helps us verify there are no deadlocks by looking only at the type's code. Otherwise, we need to understand how all the other threads are interacting.

Reply:
thanks, moved inside `GetExcludedNamespaces()`